integrations/etcd/documentation.yaml

```diff
@@ -0,0 +1,79 @@
+exporter_type: included
+app_name_short: Etcd
+app_name: {{app_name_short}}
+app_site_name: Etcd
+app_site_url: https://etcd.io/
+exporter_name: the Etcd exporter
+exporter_pkg_name: etcd
+exporter_repo_url: https://etcd.io/docs/v3.1/op-guide/monitoring/
+additional_prereq_info: |
+  {{app_site_name}} exposes Prometheus-format metrics automatically; you do not
+  have to install it separately. To verify that {{exporter_name}} is emitting 
+  metrics on the expected endpoints, do the following:
+
+  1. Set up port forwarding by using the following command:
+
+  <pre class="devsite-click-to-copy">
+  kubectl -n {{namespace_name}} port-forward {{pod_name}} 2379
+  </pre>
+
+  2. Access the endpoint `localhost:2379/metrics` by using the browser
+  or the `curl` utility in another terminal session.
+dashboard_available: true
+multiple_dashboards: false
+dashboard_display_name: {{app_name_short}} Prometheus Overview
+podmonitoring_config: |
+  apiVersion: monitoring.googleapis.com/v1
+  kind: PodMonitoring
+  metadata:
+    name: etcd
+    labels:
+      app.kubernetes.io/name: etcd
+      app.kubernetes.io/part-of: google-cloud-managed-prometheus
+  spec:
+    endpoints:
+  + - port: 2379
+      scheme: http
+      interval: 30s
+      path: /metrics
+    selector:
+      matchLabels:
+  +     app.kubernetes.io/name: etcd
```

Comment:
No "+"s needed in podmonitoring config because we're asking users to install one from scratch, not make changes to an existing setup.

Reply:
Additionally, it seems like there are many ways to deploy etcd which result in different labels.

For example, I followed this sample yaml - https://github.com/etcd-io/etcd/tree/main/hack/kubernetes-deploy - which used `app: etcd`, so we can't assume all etcd deployments will have `app.kubernetes.io/name: etcd`.

Let's add a paragraph to `additional_podmonitoring_info` reminding users to double-check that the labels match the selector, in the same style as [this](https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/blob/master/integrations/rabbitmq/documentation.yaml#L44), especially given that this is an `exporter_type: included` doc, and we are assuming they already have an etcd deployment up and running, and are not providing any sample yaml for it.